test/api_integration/apis/saved_objects/import.js

```diff
@@ -0,0 +1,121 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import expect from 'expect.js';
+import { join } from 'path';
+
+export default function ({ getService }) {
+  const supertest = getService('supertest');
+  const esArchiver = getService('esArchiver');
+
+  describe('import', () => {
+    describe('with kibana index', () => {
+      describe('without basic data existing', () => {
+        // Cleanup data that got created in import
```

Comment:
In order for these tests to qualify as `with kibana index` we should probably load/unload the esArchives here too.

Reply:
I moved these changes under `with basic data existing` as the tests didn't mind if data existed. One question is for `resolve_import_conflicts.js` file, for tests that don't want any existing data, should they move under `without kibana index`?